psalm.xml.dist

```diff
@@ -1,10 +1,9 @@
 <?xml version="1.0"?>
 <psalm
-        totallyTyped="true"
+        errorLevel="1"
```

Comment:
I'd suggest to keep the previous configuration so that new code follows a more strict setup.

Reply:
This is the most strictest level. 
totallyTyped option is deprecated.

Reply:
TIL!